AllJoyn/Platform/Bridge/BridgeCore/AllJoynHelper.cpp

```diff
@@ -1242,6 +1359,10 @@ QStatus AllJoynHelper::GetSignature(_In_ PropertyType propertyType, _Out_ std::s
         signature = "as";
         break;
 
+    case PropertyType::StringDictionary:
+        signature = "a{ss}";
```

Comment:
d may indicate double..But any other.. m ? ( map )

---
In reply to: [67949261](https://github.com/tjaffri/msiot-samples/pull/7#discussion_r67949261) [](ancestors = 67949261)